src/core/qgsdiagramrendererv2.h

```diff
@@ -341,6 +343,8 @@ class CORE_EXPORT QgsDiagramSettings
     QList< QString > categoryLabels;
     QSizeF size; //size
     SizeType sizeType; //mm or map units
+    QgsSymbolV2::OutputUnit lineSizeType; //! line unit index (mm, map units, or pixels)
```

Comment:
Not quite right - the //! must appear above the declaration:

```
//! line unit index (mm, map units, or pixels)
QgsSymbolV2::OutputUnit lineSizeType;
```

Otherwise it applies to the following line. That's how Doxygen works